SECP-service/src/main/java/com/visucius/secp/models/User.java

```diff
@@ -82,15 +86,19 @@
     @Column(name = "login_role", nullable = false)
     private LoginRole loginRole = LoginRole.NORMAL;
 
-    public  User () {
+    @Column(name = "isActive", nullable = false)
+    private boolean isActive;
```

Comment:
setting isActive to true here didn't work? it saves you from duplicating in the constructor